src/starknet_cli_wrapper.py

```diff
@@ -30,13 +31,26 @@ async def starknet_compile_main_wrapper(args):
         print(err, file=sys.stderr)
         return 1
 
+async def cargo_run(args):
+    sys.argv = [sys.argv[0], *args]
+    try:
+        args_split = " ".join(args).split("&&")
```

Comment:
Hey @FabijanC, Is this a sensible approach, or would you recommend something else here?

Reply:
What is `&&`?

Reply:
It's a way I used to separate the two commands `starknet-compile` and `starknet-sierra-compile` and execute them separately.